app/controllers/dashboard_controller.rb

```diff
@@ -1,3 +1,5 @@
 class DashboardController < ApplicationController
-  def index; end
+  def index
+    @user = User.first
+  end
```

Comment:
I would recommend using "current_user" instead of User.first since that just pulls the first user in the Users table

Reply:
Resolved